asv_bench/benchmarks/io/parquet.py

```diff
@@ -0,0 +1,58 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import modin.pandas as pd
+
+from ..utils import (
+    ASV_USE_IMPL,
+    IMPL,
+    execute,
+    get_shape_id,
+    prepare_io_data_parquet,
+    get_benchmark_shapes,
+)
+
+
+class BaseReadParquet:
+    # test data file should be created only once
+    def setup_cache(self, test_filename="io_test_file"):
+        test_filenames = prepare_io_data_parquet(
+            test_filename, self.data_type, get_benchmark_shapes(self.__class__.__name__)
+        )
+        return test_filenames
+
+    def setup(self, test_filenames, shape, *args, **kwargs):
+        # ray init
+        if ASV_USE_IMPL == "modin":
+            pd.DataFrame([])
+        self.shape_id = get_shape_id(shape)
+
+
+class TimeReadParquet(BaseReadParquet):
+    shapes = get_benchmark_shapes("TimeReadParquet")
+    data_type = "str_int"
```

Comment:
should data_type here include `{"int", "true_false_int"}` as well (from prepare_io_data_parquet).

Reply:
I don't think it's necessary to test the case where all data is numeric, since the `str_int` data type is more general - it includes both string columns and numeric. There is no specificity in our reading algorithms in these case.

As for `true_false_int` data type, we should write a separate benchmark (for example, like `TimeReadCsvTrueFalseValues`), but I would not add it until we stop falling into pandas when `true_values` or `false_values` parameters are specified.

Reply:
With `read_csv` in Pandas there are parameters to specify `true_values` and `false_values` that we don't have with `read_sql`. Good point on the `data_type` comment with ensure consistency with `prepare_io_data_parquet`.